### PR TITLE
Update documentation after changing the RemoteEnvironment CRD

### PR DESCRIPTION
**Pull request by mszostok**
**Description**

Changes proposed in this pull request:

- Update documentation after changing the RemoteEnvironment CRD

**Related issue(s)**
- https://github.com/kyma-project/kyma/issues/723

### CODE REVIEW

File: docs/application-connector/docs/040-remote-evironment-custom-resource.md [hunk 1]
```diff
@@ -54,16 +52,15 @@ This table lists all the possible parameters of a given resource together with t
...
-| **spec.source** |    **NO**   | Identifies the Remote Environment in the cluster. |
-| **spec.source.environment** |    **YES**   | Specifies the environment of the connected Remote Environment. |
-| **spec.source.type** |    **YES**   | Specifies the type of the connected Remote Environment. |
-| **spec.source.namespace** |    **YES**   | Specifies the namespace of the connected Remote Environment. |
 | **spec.description** |    **NO**   | Describes the connected Remote Environment.  |
 | **spec.accessLabel** |    **NO**   | Labels the environment when the [EnvironmentMapping](041-environment-mapping-custom-resource.md) is created. |
 | **spec.services** |    **NO**   | Contains all services that the Remote Environment provides. |
 | **spec.services.id** |    **YES**   | Identifies the service that the Remote Environment provides. |
+| **spec.services.name** |    **YES**   | Specifies the CLI-friendly name of the Remote Environment service. MUST only contain lowercase characters, numbers and hyphens (no spaces). MUST be unique across all service objects returned in Remote Environment custom resource. MUST be a non-empty string. |
```
Comment (klaudiagrz) [inline docs/application-connector/docs/040-remote-evironment-custom-resource.md]: ``Specifies the CLI-friendly name of the Remote Environment service. It must contain only lowercase characters, numbers, and hyphens, with no spaces. This field must be unique across all service objects returned in the Remote Environment custom resource. This string cannot be empty.``

File: docs/application-connector/docs/040-remote-evironment-custom-resource.md [hunk 2]
```diff
@@ -54,16 +52,15 @@ This table lists all the possible parameters of a given resource together with t
...
 | **spec.services.displayName** |    **YES**   | Specifies a human-readable name of the Remote Environment service. |
-| **spec.services.longDescription** |    **NO**   | Provides a human-readable description of the Remote Environment service. |
+| **spec.services.description** |    **YES**   | Specifies the short description of the Remote Environment service. MUST be a non-empty string. |
```
Comment (klaudiagrz) [inline docs/application-connector/docs/040-remote-evironment-custom-resource.md]: `Provides a short description of the Remote Environment service. This string cannot be empty.`

File: docs/application-connector/docs/040-remote-evironment-custom-resource.md [hunk 3]
```diff
@@ -54,16 +52,15 @@ This table lists all the possible parameters of a given resource together with t
...
+| **spec.services.labels** |    **NO**   | Specifies a labels used to add additional meta-information to the Remote Environment service. MUST contain `connected-app` label. |
```
Comment (klaudiagrz) [inline docs/application-connector/docs/040-remote-evironment-custom-resource.md]: ``Specifies labels used to add the additional meta-information to the Remote Environment service. This field must contain the `connected-app` label.``

File: docs/application-connector/docs/040-remote-evironment-custom-resource.md [hunk 4]
```diff
@@ -54,16 +52,15 @@ This table lists all the possible parameters of a given resource together with t
...
+| **spec.services.longDescription** |    **NO**   | Provides a detailed human-readable description of the Remote Environment service. |
```
Comment (klaudiagrz) [inline docs/application-connector/docs/040-remote-evironment-custom-resource.md]: ``Provides a detailed, human-readable``

File: docs/service-brokers/docs/040-event-activation-custom-resource.md [hunk 5]
```diff
@@ -37,7 +34,4 @@ This table lists all the possible parameters of a given resource together with t
 | **metadata.name** |    **YES**   | Specifies the name of the CR and the ID of the Remote Environment service. This field is also used to fetch Event schemas from the Minio storage.  |
 | **metadata.namespace** |    **YES**   | Specifies the Namespace in which the CR is created. |
 | **spec.displayName** |    **YES**   | Specifies a human-readable name of the Remote Environment service. |
-| **spec.source** |    **YES**   | Identifies a Remote Environment in the cluster. |
-| **spec.source.environment** |    **YES**   | Specifies the environment of the connected Remote Environment. |
-| **spec.source.type** |    **YES**   | Specifies the type of the connected Remote Environment. |
-| **spec.source.namespace** |    **YES**   | Specifies the namespace of the connected Remote Environment. |
+| **spec.sourceId** |    **YES**   | TBD. |
```
Comment (klaudiagrz) [inline docs/service-brokers/docs/040-event-activation-custom-resource.md]: ?!

File: docs/application-connector/docs/040-remote-evironment-custom-resource.md [hunk 6]
```diff
@@ -54,16 +52,15 @@ This table lists all the possible parameters of a given resource together with t
...
-| **spec.source** |    **NO**   | Identifies the Remote Environment in the cluster. |
-| **spec.source.environment** |    **YES**   | Specifies the environment of the connected Remote Environment. |
-| **spec.source.type** |    **YES**   | Specifies the type of the connected Remote Environment. |
-| **spec.source.namespace** |    **YES**   | Specifies the namespace of the connected Remote Environment. |
 | **spec.description** |    **NO**   | Describes the connected Remote Environment.  |
 | **spec.accessLabel** |    **NO**   | Labels the environment when the [EnvironmentMapping](041-environment-mapping-custom-resource.md) is created. |
 | **spec.services** |    **NO**   | Contains all services that the Remote Environment provides. |
 | **spec.services.id** |    **YES**   | Identifies the service that the Remote Environment provides. |
+| **spec.services.name** |    **YES**   | Specifies the CLI-friendly name of the Remote Environment service. It must contain only lowercase characters, numbers, and hyphens, with no spaces. This field must be unique across all service objects returned in the Remote Environment custom resource. This string cannot be empty. |
```
Comment (klaudiagrz) [inline docs/application-connector/docs/040-remote-evironment-custom-resource.md]: ~This string cannot be empty.~

File: docs/application-connector/docs/040-remote-evironment-custom-resource.md [hunk 7]
```diff
@@ -54,16 +52,15 @@ This table lists all the possible parameters of a given resource together with t
...
 | **spec.services.displayName** |    **YES**   | Specifies a human-readable name of the Remote Environment service. |
-| **spec.services.longDescription** |    **NO**   | Provides a human-readable description of the Remote Environment service. |
+| **spec.services.description** |    **YES**   | Provides a short description of the Remote Environment service. This string cannot be empty. |
```
Comment (klaudiagrz) [inline docs/application-connector/docs/040-remote-evironment-custom-resource.md]: ~This string cannot be empty.~

File: docs/application-connector/docs/040-remote-evironment-custom-resource.md [hunk 8]
```diff
@@ -54,16 +52,15 @@ This table lists all the possible parameters of a given resource together with t
...
+| **spec.services.labels** |    **NO**   | Specifies labels used to add the additional meta-information to the Remote Environment service. This field must contain the `connected-app` label. |
+| **spec.services.longDescription** |    **NO**   | Provides a detailed, human-readable. |
```
Comment (klaudiagrz) [inline docs/application-connector/docs/040-remote-evironment-custom-resource.md]: Please end this sentence 😆

File: components/remote-environment-broker/contrib/examples/remote-env.yaml [hunk 9]
```diff
@@ -1,35 +1,33 @@
-apiVersion: remoteenvironment.kyma.cx/v1alpha1
+apiVersion: applicationconnector.kyma-project.io/v1alpha1
 kind: RemoteEnvironment
 metadata:
   name: ec-prod
 spec:
   accessLabel: access-label-1
-
-  # source identifies remote environment in the cluster
```
Comment (aszecowka) [inline components/remote-environment-broker/contrib/examples/remote-env.yaml]: should we have here sourceId?

**Pull request merged**